README.md

```diff
@@ -4,75 +4,46 @@
 
 ![Screenshot of the Gutenberg Editor, editing a post in WordPress](https://cldup.com/R84R5fNgrI.png)
 
-This repo is the development hub for the <a href="https://make.wordpress.org/core/2017/01/04/focus-tech-and-design-leads/">editor focus in WordPress Core</a>. `Gutenberg` is the project name.
+Welcome to the development hub for the WordPress Gutenberg Project!
 
-## Getting started
-- **Download:** If you want to use the latest release with your WordPress site, <a href="https://wordpress.org/plugins/gutenberg/">download the latest release from the WordPress.org plugins repository</a>.
-- **Discuss:** Conversations and discussions take place in <a href="https://wordpress.slack.com/messages/C02QB2JS7">`#core-editor` channel on the Making WordPress Slack</a>.
-- **Contribute:** Development of Gutenberg happens in this GitHub repo. Get started by <a href="https://github.com/WordPress/gutenberg/blob/master/CONTRIBUTING.md">reading the contributing guidelines</a>.
-- **Learn:** <a href="https://wordpress.org/gutenberg/">Discover more about the project on WordPress.org</a>.
-
-**Gutenberg is more than an editor.** While the project is currently focused on building the new editor for WordPress, it doesn't end there. This lays the groundwork for a new model for WordPress Core that will ultimately impact the entire publishing experience of the platform.
-
-## Editing focus
-
-> *The editor will create a new page- and post-building experience that makes writing rich posts effortless, and has “blocks” to make it easy what today might take shortcodes, custom HTML, or “mystery meat” embed discovery.*
->
-> — Matt Mullenweg
+Gutenberg is more than an editor. It is the foundation that will revolutionize customization, site building and the entire publishing experience of the platform. The four phases of the project are Editing, Customization, Collaboration, and Multilingual.
```

Comment:
This feels a little grand, honestly. It's definitely the intent, and our ultimate hope, but have we achieved it? Not yet, and let's not jinx it.

Also, as a followup paragraph to a heading including the term "Gutenberg Project", this is an opportunity for us to explain that "Gutenberg" is a **codename** for the project that is being delivered in plugin form, and that the final result is merged into WordPress under the term **block editor**.

Reply:
Will use Michelle's contrib below, thanks for the reviews :+1: